src/sql.c

```diff
@@ -0,0 +1,5 @@
+#include "common.h"
+
+int main(void) {
```

Comment:
You will probably have to update this file again too with the changes you currently have.